src/api/index.ts

```diff
@@ -35,4 +35,22 @@ apiClient.interceptors.response.use(
   }
 );
 
+export async function loginApi(email: string, password: string) {
+  const response = await axios.post(`${baseURL}/api/login`, {
```

Comment:
baseurl 설정이 담긴 공용 apiClient 객체를 위에서 만들어주신게 있어서 axios 대신 apiClient를 쓰면 baseUrl 지정 필요 없이 아래와 같이 작성이 가능합니다! 

`axios.create`는 만드신 것 처럼 api 호출시 필요한 공통 세팅(ex-baseurl)을 담은 axios 객체를 미리 인스턴스로 만들어두어 반복적인 config 세팅을 하지 않도록 도와줍니다.

```suggestion
  const response = await apiClient.post(`/api/login`, {
```